handlers/mesh.go

```diff
@@ -46,6 +50,7 @@ func MeshGraph(
 		o := mesh.NewOptions(r, &business.Namespace)
 
 		meshInfo, err := discovery.Mesh(r.Context())
+		filterAccessibleControlPlanes(r.Context(), business.Namespace, meshInfo)
```

Comment:
If you try to hit this endpoint as the `bookinfo` user you get a `500` response. Probably need to return a 403 response when we get back a 403 response from every `GetClusterNamespace` check. Reproduce with:

1. Setup multi-primary

    ```
    hack/run-integration-tests.sh --test-suite frontend-multi-primary --setup-only true
    ```
2. Login as bookinfo user and grab cookie from web inspector
3. Make api request with cookie to graph the mesh page
    ```
    curl -v --cookie kiali-token-east=<cookie> <kiali_url/api/mesh/graph
    ```
This is just an edge case though since we hide the mesh page if you don't have access to any controlplane namespaces. Right now the only way to hit this error is to use the api directly outside the UI.

Reply:
Yes, it can be reproduced also with the token strategy (See description) with a token for the test SA. 
I have updated the logic, to return 403 in this case.